custom_model_runner/datarobot_drum/drum/artifact_predictors/sklearn_predictor.py

```diff
@@ -69,6 +70,13 @@ def predict(self, data, model, **kwargs):
             else:
                 labels_to_use = self.class_labels
             predictions = model.predict_proba(data)
+            if predictions.shape[1] == 1:
```

Comment:
is it possible to do a better assertion than this, to asses that hte shape is the right shape?

Reply:
sure